### PR TITLE
Enable sorting for roles, operators and rejections

### PR DESCRIPTION
**Pull request by kamsi**
IT corresponds to following Jira tickets:

https://unctad.atlassian.net/browse/EV2CORE-107
https://unctad.atlassian.net/browse/EV2CORE-108
https://unctad.atlassian.net/browse/EV2CORE-109

### CODE REVIEW

File: view/utils/init-table-sorting-on-client.js [hunk 1]
```diff
@@ -16,6 +16,23 @@ module.exports = function (domElement/*, opts */) {
 		// set type, either numeric or text 
 		type: 'numeric'
 	});
+	window.jQuery.tablesorter.addParser({
+		// set a unique id 
+		id: 'dates',
+		is: function (s) {
+			// return false so this parser is not auto detected
```
Comment (mtuchowski) [inline view/utils/init-table-sorting-on-client.js]: While stating that returning false will prevent auto-detection is valuable information I would like to know why we don't want it to be auto-detected as well...

File: view/utils/init-table-sorting-on-client.js [hunk 2]
```diff
@@ -16,6 +16,23 @@ module.exports = function (domElement/*, opts */) {
...
+		// set a unique id 
+		id: 'dates',
+		is: function (s) {
+			// return false so this parser is not auto detected 
+			return false;
+		},
+		format: function (item) {
+			var parsedItem = item;
+			if (item.indexOf('/') !== -1) {
+				parsedItem = item.replace(/\/+/g, '-').split('-').reverse().join('-');
```
Comment (mtuchowski) [inline view/utils/init-table-sorting-on-client.js]: Don't we have any locale dependent formatter for dates in JS/our system?

**Comment by mtuchowski** [comment]
@kamsi Please, prepare test port and update JIRA tickets with those.

File: view/utils/init-table-sorting-on-client.js (hunk 1 above)
Comment (kamsi) [inline view/utils/init-table-sorting-on-client.js]: @mtuchowski not sure about internals, but to my understanding this assures that we will not use it by accident (which inmo is undesired).

File: view/utils/init-table-sorting-on-client.js (hunk 2 above)
Comment (kamsi) [inline view/utils/init-table-sorting-on-client.js]: @mtuchowski brief search did not bring anything interesting. Still it didn't work ouside of the box for dbjs Date... even though it stringified itself to that...

**Comment by kamsi** [comment]
@mtuchowski it passed the tests, so I think we can merge.

**Comment by mtuchowski** [comment]
@kamsi Please resolve conflicts.

**Pull request merged**